webapp/channels/src/components/channel_header_dropdown/menu_items/close_channel/close_channel.tsx

```diff
@@ -7,25 +7,31 @@ import Menu from 'components/widgets/menu/menu';
 
 import {localizeMessage} from 'utils/utils';
 
-type Props = {
+interface CloseChannelProps {
```

Comment:
The name change is not needed. There is only one component in this file, and the props are only used in this file, so it is inferred that the props are of the CloseChannel component.